apps/condo/domains/banking/schema/BankTransaction.test.js

```diff
@@ -0,0 +1,557 @@
+                const [integrationContext] = await createTestBankIntegrationContext(admin, bankIntegration, organization)
+                const [account] = await createTestBankAccount(admin, organization, property, {
+                    integrationContext: { connect: { id: integrationContext.id } },
+                })
+                const [contractorAccount] = await createTestBankContractorAccount(support, organization)
+                const [obj, attrs] = await createTestBankTransaction(admin, account, contractorAccount, integrationContext, organization)
+
+                expect(obj.id).toMatch(UUID_RE)
+                expect(obj.dv).toEqual(1)
+                expect(obj.sender).toEqual(attrs.sender)
+                expect(obj.v).toEqual(1)
+                expect(obj.newId).toEqual(null)
+                expect(obj.deletedAt).toEqual(null)
+                expect(obj.createdBy).toEqual(expect.objectContaining({ id: admin.user.id }))
+                expect(obj.updatedBy).toEqual(expect.objectContaining({ id: admin.user.id }))
+                expect(obj.createdAt).toMatch(DATETIME_RE)
+                expect(obj.updatedAt).toMatch(DATETIME_RE)
+                expect(obj.number).toEqual(attrs.number)
+                expect(obj.date).toEqual(attrs.date)
+                expect(parseFloat(obj.amount)).toBeCloseTo(parseFloat(attrs.amount), 2)
+                expect(obj.currencyCode).toEqual(attrs.currencyCode)
+                expect(obj.purpose).toEqual(attrs.purpose)
+                expect(obj.dateWithdrawed).toEqual(attrs.dateWithdrawed)
+                expect(obj.importId).toEqual(attrs.importId)
+                expect(obj.importRemoteSystem).toEqual(attrs.importRemoteSystem)
+                expect(obj.meta).toEqual(attrs.meta)
+                expect(obj.integrationContext).toMatchObject(pick(integrationContext, ['id', 'enabled']))
+            })
+
+            test('support can\'t', async () => {
+                const [organization] = await createTestOrganization(admin)
+                const [property] = await createTestProperty(admin, organization)
+                const [integrationContext] = await createTestBankIntegrationContext(admin, bankIntegration, organization)
+                const [account] = await createTestBankAccount(admin, organization, property, {
+                    integrationContext: { connect: { id: integrationContext.id } },
+                })
+                const [contractorAccount] = await createTestBankContractorAccount(support, organization)
+
+                await expectToThrowAccessDeniedErrorToObj(async () => {
+                    await createTestBankTransaction(support, account, contractorAccount, integrationContext, organization)
+                })
+            })
+
+            test('user can if it is an employee of organization with "canManageBankTransactions" permission', async () => {
+                const userClient = await makeClientWithNewRegisteredAndLoggedInUser()
```

Comment:
Perhaps it's better to put it in `beforeAll`. Each test creates a new `organization` and `employee` instance. It seems that in this case there will be no collisions in different tests when using the same user and different organizations

Reply:
Same for tests below

Reply:
Yes, it is better to create an organization and other entities related to it clean. I am looking at userClient. It seems that userClient can be the same in different organizations.